.goreleaser.yml

```diff
@@ -369,6 +413,21 @@ dockers:
       - release_deps/opentelemetry-java-contrib-jmx-metrics.jar
 
 docker_manifests:
```

Comment:
This section cannot be tested unless we issue a release. We should ensure the image_template names line up with the image names used in the `dockers` step.